File: Python/Niveau 1/7 – Conditions avancées, opérateurs booléens/Maison de l'espion.py
xmin = int(input())
xmax = int(input())
ymin = int(input())
ymax = int(input())
maisons = int(input())
tot = 0

for i in range(maisons):
   xm = int(input())
   ym = int(input())
   if xmin <= xm  <= xmax and ymin <= ym <= ymax: 
         tot += 1

print(tot)

File: Pascal/Niveau 1/7 – Conditions avancées, opérateurs booléens/Maison de l'espion.pas
program Solution;
var
   xMin, xMax, yMin, yMax, houses, dig, x, y, i: LongInt;
begin
   read(xMin, xMax, yMin, yMax, houses);
   dig := 0;
   for i := 1 to houses do
   begin
      read(x, y);
      if (xMin <= x) and (x <= xMax) and (yMin <= y) and (y <= yMax) then
      begin
         dig := dig + 1;
      end;
   end;
   writeln(dig);
end.